code_examples/sdk_examples/src/core_features/messaging/03_decrypt_message.ts

```diff
@@ -0,0 +1,36 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { useDecryptionSignCallback } from '../signCallback/useDecryptionSignCallback'
+
+export async function decryptMessage(
+  encryptedMessage: Kilt.IEncryptedMessage,
+  keyAgreement: Kilt.KiltEncryptionKeypair
+): Promise<void> {
```

Comment:
This function would typically return the decrypted message, not simply `Promise<void>`, right?

Reply:
Yes, it would normally return the decrypted Message. I can change it to return the `decrypedMessage`.